lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -3,38 +3,47 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
```

Comment:
Looks like some new lines are automatically added by your IDE, please avoid this.